app/boards/acp_6_0_adsp.conf

```diff
@@ -5,7 +5,6 @@ CONFIG_DCACHE_LINE_SIZE=128
 CONFIG_DYNAMIC_INTERRUPTS=y
```

Comment:
@andyross Ack. CONFIG_TRACE was working with Zephyr at some point, but basicly as Intel has moved to native logging, this code is likely to have bitrotted. For Intel, we are ok to remove it. It was useful at start, but now that there is plenty of infra available in Zephyr (and plenty of examples in SOF how to use it), I don't think it's the worth keeping around anymore. And if it doesn't work anymore, seems like a clear case.